abstracts/abstract_pl.tex

```diff
@@ -8,10 +8,15 @@
     Proceduralne generowanie świata jest techniką zdobywającą rosnącą popularność przy tworzeniu gier komputerowych.
     Zostało ono wykorzystane z powodzeniem m. in. w takich produkcjach jak \textit{Minecraft} czy \textit{No Man's Sky}.
     Z drugiej strony za przyczyną gier takich jak \textit{Hyperbolica}, szersze zainteresowanie zyskała kwestia osadzania gier komputerowych w przestrzeniach nieeuklidesowych.
-    Za cel niniejszej pracy obrano stworzenie aplikacji graficznej łączącej obydwa wspomniane elementy.\\
+    Niniejsza praca opisuje implementację gry komputerowej, która łączy oba te podejścia.
 
-    \todo{Translate from english version when that one is finished.}
+    Gra przekształca przestrzeń euklidesową w przestrzeń hiperboliczną lub sferyczną, co pozwala na wykorzystanie istniejących algorytmów.
```

Comment:
Idk if we shouldn't change that (in english version as well) to "W grze zastosowano metodę przekształcania przestrzeni euklidesowej ..." so in english "In the game we used the technique of ..."
But that's a detail